app/Resources/views/admin/blog/_form.html.twig

```diff
@@ -6,7 +6,11 @@
     {{ form_start(form, { attr: { novalidate: 'novalidate' } }) }}
 #}
 
-{{ form_start(form) }}
+{% if show_confirmation_message is defined and show_confirmation_message == true %}
```

Comment:
@yceruto sorry for adding a new comment in this pull request which you already worked on a lot. I'm wondering if we could replace this long condition by just:

```twig
{% if show_confirmation_message|default(false) %}
```

I'm just asking. I don't really know which is the common practice in these cases.